src/apps/service/apps/combined/util.clj

```diff
@@ -1,5 +1,5 @@
 (ns apps.service.apps.combined.util
-  (:use [apps.service.util :only [sort-apps apply-offset apply-limit uuid?]]
+  (:use [apps.service.util :only [sort-apps apply-offset apply-limit valid-uuid?]]
```

Comment:
I don't think `valid-uuid?` is used anywhere in this file.

Reply:
I'll remove that too. Thanks!